### PR TITLE
format and zzz

### PR DESCRIPTION
**Pull request by randrescastaneda**
Hi Ronak, 

I made a few unimportant updates to the auto_aux_update(). They just make the file a little easier to read. However, the changes to zzz.R are important because we need some of the information in the options to make other functions to work. For example, we need `getOption("pipaux.cpivar")` to make `pip_cpi_clean()` to work. I commented the creation of `gls` though` , so it does not mess up with Posit connect. We should chat a little about this. 

thanks.

### CODE REVIEW

**Comment by randrescastaneda** [comment]
Hi @shahronak47 , 

One more thing. The `auto_aux_update()` function is working but it seems to me that it is not update the metdata file properly. My understanding is that if it runs once and updates all the necessary aux file, the next time I run it should not display anything because everything has been updated already. Yet, take a look at the image below where I executed the function one time after the other and it displayed the same info. 

image

File: R/auto_aux_update.R [hunk 1]
```diff
@@ -83,21 +94,30 @@ auto_aux_update <- function(measure = NULL,
...
+            file_path = "Data/git_metadata.csv",
+            .params   = list(ref = "metadata"))
+
+  res <- gh::gh("PUT /repos/{owner}/{repo}/contents/{path}",
+                owner   = "PIP-Technical-Team",
+                repo    = "pipaux",
+                path    = "Data/git_metadata.csv",
+                .params = list(branch  = "metadata",
+                               message = "updating csv file",
+                               sha     = out$sha, # why does the sha remain the same?
```
Comment (shahronak47) [inline R/auto_aux_update.R:114]: `sha` does not remain the same but we get it from the above `GET` command

**Comment by shahronak47** [comment]
Hi @randrescastaneda , how are you reproducing this?

**Comment by randrescastaneda** [comment]
> Hi @randrescastaneda , how are you reproducing this?

@shahronak47 , could you please expand a little on this question? I am just working from the package project directly, loading it with `devtools::load_all()`. Is that what you mean?

**Comment by randrescastaneda** [comment]
Hi @shahronak47 , 

There is definitely something weird. Look at the image below. I executed the function, but it did not find the change in `aux_censoring` repo. I know the data changed, so I subsequently executed the `pip_censoring()` function and then the data got updated. What do you think the problem is?

Thanks. 

image

**Comment by shahronak47** [comment]
> Hi @shahronak47 ,
> 
> There is definitely something weird. Look at the image below. I executed the function, but it did not find the change in `aux_censoring` repo. I know the data changed, so I subsequently executed the `pip_censoring()` function and then the data got updated. What do you think the problem is?
> 
> Thanks.
> 
> image

Ok. Let's discuss this and other issues on call. I'll send you a meeting request for Monday.

**Comment by shahronak47** [comment]
> Hi @shahronak47 ,
> 
> There is definitely something weird. Look at the image below. I executed the function, but it did not find the change in `aux_censoring` repo. I know the data changed, so I subsequently executed the `pip_censoring()` function and then the data got updated. What do you think the problem is?
> 
> Thanks.
> 
> image

I checked this and realized that censoring was not part of the yml file. I have added it now and it should work.

**Comment by shahronak47** [comment]
> Hi @shahronak47 ,
> 
> One more thing. The `auto_aux_update()` function is working but it seems to me that it is not update the metdata file properly. My understanding is that if it runs once and updates all the necessary aux file, the next time I run it should not display anything because everything has been updated already. Yet, take a look at the image below where I executed the function one time after the other and it displayed the same info.
> 
> image

This is because there is a cached version of the older file in the API response. If you wait for a minute or so after running the first time and run the function again, it will not update the same files again.

**Pull request merged**